tea.go

```diff
@@ -1050,6 +1050,10 @@ func (p *Program) recoverFromPanic() {
 // ReleaseTerminal restores the original terminal state and cancels the input
 // reader. You can return control to the Program with RestoreTerminal.
 func (p *Program) ReleaseTerminal() error {
+	return p.releaseTerminal(false)
```

Comment:
I'm not sure if this is the right place to have the reset flag. Perhaps we should change the signature of `p.RestoreTerminal(reset bool) error`